server-spi/src/main/java/org/keycloak/models/RealmModel.java

```diff
@@ -267,6 +267,9 @@ default Boolean getAttribute(String name, Boolean defaultValue) {
     int getActionTokenGeneratedByAdminLifespan();
     void setActionTokenGeneratedByAdminLifespan(int seconds);
 
+    int getRequestUriLifespan();
```

Comment:
It will be good if we can avoid changes in the RealmModel, RealmAdapter, RepresentationToModel and other core model classes just because of introducing PAR related configurations. Is it possible to add some class like `ParConfig` and move the PAR related configuration parameters to this class? See `CibaConfig` for the inspiration. Note that it has some realm-scoped configuration attributes as well as client-scoped configuration attributes and works fine for both.

Reply:
@mposolda I see, I will follow the way of implementing newly introduced configuration in `CibaConfig` and avoid modifying `RealmModel` directly.

Reply:
Thanks